code/modules/mob/living/simple_animal/hostile/megafauna/dragon.dm

```diff
@@ -41,6 +42,13 @@
 		return
 	else
 		..()
+		if(isliving(target))
+			var/mob/living/L = target
+			if(L.stat)
```

Comment:
Make it check if `stat == DEAD`, put this block of code above ..(), and have it return, so normal ash drakes won't autogib people they crit because that's absolute shit kor